<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propertiesPanel</class>
 <widget class="QWidget" name="propertiesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="Accept">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqUpdate16.png</normaloff>:/pqWidgets/Icons/pqUpdate16.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Reset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqCancel16.png</normaloff>:/pqWidgets/Icons/pqCancel16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
         <normaloff>:/QtWidgets/Icons/pqDelete16.png</normaloff>:/QtWidgets/Icons/pqDelete16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Help">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqHelp16.png</normaloff>:/pqWidgets/Icons/pqHelp16.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SearchBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="SearchLineEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search for properties by name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Search ... (use Esc to clear text)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AdvancedButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle advanced properties&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqAdvanced26.png</normaloff>:/pqWidgets/Icons/pqAdvanced26.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>332</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ScrollAreaLayout" stretch="0,0,0,0,1">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="pqExpanderButton" name="PropertiesButton">
         <property name="text" stdset="0">
          <string>Properties</string>
         </property>
         <property name="checked" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="PropertiesFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqExpanderButton" name="DisplayButton">
         <property name="text" stdset="0">
          <string>Display</string>
         </property>
         <property name="checked" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="DisplayFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqExpanderButton</class>
   <extends>QFrame</extends>
   <header>pqExpanderButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
